Dockerfile

```diff
@@ -1,4 +1,8 @@
-FROM nikolaik/python-nodejs:latest
+FROM python:3
```

Comment:
Perhaps later we can change this image to `python:3-slim` for smaller container size. However, after some initial (and short) tests, `python:3-slim` is lacking some resources and therefore CryptoBook builds with errors.

With this current set-up, CryptoBook's container is a staggering 1.14gb with all the dependencies installed on the Docker build. 

<img width="1234" alt="Screen Shot 2019-06-11 at 5 12 00 PM" src="https://user-images.githubusercontent.com/2829082/59307216-0a610d80-8c6c-11e9-9af1-c13da79805e8.png">

Reply:
This is fixed with #20.